apps/ledger-live-desktop/src/renderer/screens/asset/BalanceSummary.tsx

```diff
@@ -90,23 +95,36 @@ export default function BalanceSummary({
       </Box>
 
       <Box px={5} ff="Inter" fontSize={4} color="palette.text.shade80" pt={6}>
-        <Chart
-          magnitude={chartMagnitude}
-          color={chartColor}
-          // TODO make date non optional
-          data={history as Data}
-          height={200}
-          tickXScale={range}
-          valueKey={displayCountervalue ? "countervalue" : "value"}
-          renderTickY={
-            discreetMode
-              ? () => ""
-              : displayCountervalue
-              ? renderTickYCounterValue
-              : renderTickYCryptoValue
-          }
-          renderTooltip={renderTooltip}
-        />
+        {currency.type === "TokenCurrency" && currency.id === "vechain/vtho" ? (
+          <>
+            <Alert type="secondary" noIcon={false}>
+              <span>{t("vechain.noGraphWarning")}</span>
+            </Alert>
+            <PlaceholderChart
+              magnitude={counterValue.units[0].magnitude}
+              data={portfolio.balanceHistory}
+              tickXScale={range}
+            />
+          </>
+        ) : (
```

Comment:
Can you please make this generic, in the same way you did for LLM?

We don't want any currency or token specific code outside of the `families` folder. What you did for mobile with a `tokensWithUnsupportedGraph` is acceptable though.

Reply:
fixed thanks